content/en/_index.html

```diff
@@ -10,8 +10,8 @@
 
 	</div>
 	<div class="col-md-7 col-lg-8 order-xs-2 order-sm-2 order-md-2">
-		<h1>cert-manager</h1>
-		<p class="lead mt-2">X.509 certificate management for Kubernetes</p>
+		<p class="lead mt-2 display-4">Cloud-native certificate management</p>
+		<p class="lead mt-2 display-5">Securely issue Kubernetes certs in minutes</p>
```

Comment:
Nit: For these two lines, I guess I somewhat prefer the original wording. Alternatively, perhaps we could use another term instead of 'Kubernetes certs' and go for 'Cloud-native' without dash?

Reply:
I struggled with this bit for sure. The first line `<h1>cert-manager</h1>` seemed redundant since "cert-manager" is written just above it and it's not providing any new information:

![cm](https://user-images.githubusercontent.com/1972547/122929675-1f94ca80-d363-11eb-8c2c-f0ace0b5dc86.png)

I shied away from "X.509 certificate management for Kubernetes" because I didn't know if less-technical people might struggle with "X.509" (e.g. a manager who's been told by their engineers that they should be using cert-manager). Maybe the original is better though, in general; I'll switch the second line back.